econml/tests/test_policy_forest.py

```diff
@@ -373,6 +382,8 @@ def test_warm_start(self,):
         return
 
     def test_plotting(self,):
+        if not graphviz_works:
+            return

```

Comment:
Do this instead:
```suggestion
    @pytest.mark.skipif(not graphviz_works, reason="graphviz must be installed to test plotting")
    def test_plotting(self,):
        if not graphviz_works:
            return

```

That way it will be clear in the test results that this test is being skipped on some platforms and passes on others, rather than looking like it passes everywhere even though it's not doing anything.

Reply:
Done